src/components/variant/ui/header.tsx

```diff
@@ -23,9 +28,25 @@ export const VariantHeader = observer(({ setLayout }: Props): ReactElement => {
   const history = useHistory()
   const location = useLocation()
 
-  const genInfo = get(toJS(variantStore.variant), '[0].rows[0].cells[0][0]', '')
+  const variant = toJS(variantStore.variant)
+  const rows: IAttributeDescriptors[] = get(variant, '[0].rows', [])
+
+  const genesRow = rows.find(element => element?.name === 'genes')
+  const variantWithoutGenesName = 'None'
+  const genes = genesRow?.cells[0][0] ?? variantWithoutGenesName
+
+  const hg19Row = rows.find(element => element.name === 'hg19')
+  const hg19locus = hg19Row?.cells[0][0] ?? ''
+
+  const hg38Row = rows.find(element => element.name === 'hg38')
+  const hg38locus = hg38Row?.cells[0][0] ?? ''
+
+  // TODO: update type after implantion IDsInfo interface
+  const { meta }: any = toJS(dirinfoStore.dsinfo)
+  const hgModeValue: HgModes = meta?.modes?.[0]
+
+  const currentLocus = hgModeValue === HgModes.HG19 ? hg19locus : hg38locus
```

Comment:
maybe `currentLocus` could be a getter in `dirinfoStore`? I don't insist but it's a subject to think about

Reply:
Fixed